index.html

```diff
@@ -236,8 +236,8 @@ <h2>is this ready for use yet?</h2>
 		<p>
       kinda! there are likely some major bugs in workspacer that I have not fixed yet, 
       but it is more or less functionally complete. i am using workspacer 24/7 at home and at 
-      work at the moment, so it suites my needs (aside from the occasional hickup or crash). 
-      if you run into any problems, please open an issue!
+      work at the moment, so it suits my needs (aside from the occasional hiccup or crash). 
+      if you run into any problems, please <a href="https://github.com/workspacer/workspacer">open an issue</a>!
```

Comment:
Shouldnt this link also point to the issues sub-page? :)

Reply:
@josteink if we could create a default template for issues I think it would be beneficial.